controllers/application_controller.php

```diff
@@ -16,7 +16,7 @@ function get_home_dir() {
  * @return string
  */
 function get_home_url() {
-  return 'http://localhost/foroweb/';
+  return 'http://localhost/PI4_Foro_Web/';
 }
```

Comment:
just remember if most of the team is using the other folder name, you must stick to that